kafka_consumer/README.md

```diff
@@ -143,7 +141,6 @@ sudo service datadog-agent restart
 [13]: https://www.datadoghq.com/blog/monitoring-kafka-performance-metrics
 [14]: https://www.datadoghq.com/blog/collecting-kafka-performance-metrics
 [15]: https://www.datadoghq.com/blog/monitor-kafka-with-datadog
-[16]: https://www.datadoghq.com/product/data-streams-monitoring/
```

Comment:
since this is being removed I wonder if the other numbers should be shifted down